### PR TITLE
new lesson object relationships

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,31 +2,122 @@
...
+![uml class](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/uml_class.png)
+
+## Association
+
+
+**Association** is used to model a general relationship between objects.
+Neither object is considered a part or member of the other.
+An association is drawn using an edge between classes.  The edge may have
+have an arrow to indicate direction.  The arrow is often left off if the association
+is bi-directions.
```
Comment (kcleland0818) [inline README.md]: Maybe "bi-directional" instead?

File: README.md [hunk 2]
```diff
@@ -2,31 +2,122 @@
...
+represent software artifacts.    We can create UML diagrams to depict class structure
+and class relationships.  We draw a class as shown below.  The fields and methods may
+be omitted.  Relationships between classes are drawn using different types of edges.
+
+![uml class](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/uml_class.png)
+
+## Association
+
+
+**Association** is used to model a general relationship between objects.
```
Comment (kcleland0818) [inline README.md]: Looks like the other terms below are italicized instead of bolded. I'd do the same here to be consistent.

**Review by kcleland0818 — APPROVED** [review]
Ooh UML! Nice lesson! Maybe also tell students what kind of tool or program they can use to create UML diagrams.

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: yes, definitely!

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed

**Comment by linda-seiter** [comment]
> Ooh UML! Nice lesson! Maybe also tell students what kind of tool or program they can use to create UML diagrams.

yes, added a link to draw.io in the conclusion

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**